draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1821,18 +1827,16 @@ are defined below depending on the type of retrieval.
 
 The user agent includes stored cookies in the Cookie HTTP request header field.
 
-When the user agent generates an HTTP request, the user agent MUST NOT attach
-more than one Cookie header field.
-
 A user agent MAY omit the Cookie header field in its entirety.  For example, the
 user agent might wish to block sending cookies during "third-party" requests
 from setting cookies (see {{third-party-cookies}}).
 
 If the user agent does attach a Cookie header field to an HTTP request, the
-user agent MUST compute the cookie-string following the algorithm defined in
-{{retrieval-algorithm}}, where the retrieval's URI is the request-uri, the
-retrieval's same-site status is computed for the HTTP request as defined in
-{{same-site-requests}}, and the retrieval's type is "HTTP".
+user agent MUST generate a single cookie-string and the user agent MUST compute
```

Comment:
```
user agent MUST generate a single cookie-string and the user agent MUST compute
```
My intention here is that the cookie code within the UA is still only trying to generate a single string (to maximize compatibility), but once that string is generated it may be split by by another layer of the networking stack.

Reply:
I agree with that. Clients MUST produce a single value. Servers MUST tolerate them being split, either by actually addressing that case or by using lower-level components that guarantee they'll be properly recombined before it sees the header value.